pkg/format/error/error.go

```diff
@@ -0,0 +1,92 @@
+package error
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/puppetlabs/errawr-go/v2/pkg/encoding"
+	"github.com/puppetlabs/relay/pkg/config"
+	"github.com/puppetlabs/relay/pkg/dialog"
+	"github.com/puppetlabs/relay/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func FormatError(err error, cmd *cobra.Command) {
```

Comment:
I would love to upstream some of this to errawr itself. It's default text Stringer is... crappy...

Reply:
I won't do it now, here's this to remind us: https://tickets.puppetlabs.com/browse/PN-726